src/electron.js

```diff
@@ -0,0 +1,14 @@
+import store from './store';
+
+export default {
+  init() {
+    store.commit('app/enableElectron')
+
+    //We use the Bridge as a way to share functions between electron and vue
+    window.Bridge.statusAppend = txt => {
+      txt = '\n' + txt;
```

Comment:
prefer backticks style over assembling strings, it's cleaner and less error prone
```
  txt = `\n${txt}`;
```